src/test/java/bio/terra/integration/DataRepoFixtures.java

```diff
@@ -1968,4 +1968,25 @@ public SnapshotAccessRequestResponse createSnapshotAccessRequest(
     assertTrue("Snapshot Access Request is present", response.getResponseObject().isPresent());
     return response.getResponseObject().get();
   }
+
+  // Currently, there is no getSnapshotAccessRequest API. So this uses the enumerate endpoint,
+  // and searches through the list to find the specified snapshot access request
+  public SnapshotAccessRequestResponse getSnapshotAccessRequest(
+      TestConfiguration.User user, UUID snapshotRequestId) throws Exception {
+    DataRepoResponse<EnumerateSnapshotAccessRequest> response =
+        dataRepoClient.get(
+            user, "/api/repository/v1/snapshotAccessRequests", new TypeReference<>() {});
+    assertThat(
+        "get Snapshot Access Request job is successful",
+        response.getStatusCode(),
+        equalTo(HttpStatus.OK));
+    assertTrue("Snapshot Access Request is present", response.getResponseObject().isPresent());
+    SnapshotAccessRequestResponse snapshotAccessRequest =
+        response.getResponseObject().get().getItems().stream()
+            .filter(s -> s.getId().equals(snapshotRequestId))
+            .findFirst()
+            .orElse(null);
+    assertNotNull("Snapshot Access Request is present", snapshotAccessRequest);
+    return snapshotAccessRequest;
```

Comment:
Instead of `orElse(null) ... assertNotNull()` I would use `orElseThrow()`
```suggestion
    return
        response.getResponseObject().get().getItems().stream()
            .filter(s -> s.getId().equals(snapshotRequestId))
            .findFirst()
            .orElseThrow();
```